index.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs/promises');
+
+const sumNumbers = async (fileA, fileB) => {
+    try {
+        const dataA = await fs.readFile(fileA, 'utf-8');
```

Comment:
You can use path.resolve to resolve file path

Reply:
+